client/app/dedicatedCloud/datacenter/dedicatedCloud-datacenter.controller.js

```diff
@@ -120,6 +143,39 @@ angular
           }));
     }
 
+    checkForZertoOptionOrder() {
+      return this.dedicatedCloudDrp.checkForZertoOptionOrder(this.currentService.name)
+        .then((storedDrpInformations) => {
+          let storedDrpStatus = this.DRP_STATUS.disabled;
+          if (storedDrpInformations) {
+            storedDrpStatus = this.dedicatedCloudDrp.constructor
+              .formatStatus(storedDrpInformations.status);
+          }
+
+          this.drpStatus = [this.currentDrp.state, storedDrpStatus]
+            .find(status => status !== this.DRP_STATUS.disabled)
+            || this.DRP_STATUS.disabled;
+
+          this.drpRemotePccStatus = this.currentDrp
+            .drpType === DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
+            ? this.dedicatedCloudDrp.constructor.formatStatus(_.get(this.currentDrp, 'remoteSiteInformation.state'))
+            : this.DRP_STATUS.delivered;
+        })
+        .catch((error) => {
+          this.loadingError = true;
+          this.$scope.setMessage(`${this.$translate.instant('dedicatedCloud_datacenter_drp_get_state_error')} ${_.get(error, 'data.message', error.message)}`);
```

Comment:
Could we avoid going through `$scope` ?

Reply:
I would like to, but this requires to make a call to `Alerter` into a `$timeout` with at least 3 seconds of delay. I know I've already done this in other states, but it bothers me to do it again.

The better should be to refactor PCC and Datacenter to remove all `$scope` at once, or even rewrite the two controllers.